crates/runc-shim/src/task.rs

```diff
@@ -95,15 +100,27 @@ where
 }
 
 impl<F, C> TaskService<F, C> {
-    pub async fn get_container(&self, id: &str) -> TtrpcResult<MappedMutexGuard<'_, C>> {
-        let mut containers = self.containers.lock().await;
+    pub async fn get_container_mut(&self, id: &str) -> TtrpcResult<RwLockMappedWriteGuard<'_, C>> {
```

Comment:
```suggestion
    pub async fn container_mut(&self, id: &str) -> TtrpcResult<RwLockMappedWriteGuard<'_, C>> {
```

Reply:
but the follow is 
The get naming is used only when there is a single and obvious thing that could reasonably be gotten by a getter. For example [Cell::get](https://doc.rust-lang.org/std/cell/struct.Cell.html#method.get) accesses the content of a Cell.

For getters that do runtime validation such as bounds checking, consider adding unsafe _unchecked variants. Typically those will have the following signatures.

> fn get(&self, index: K) -> Option<&V>;
> fn get_mut(&mut self, index: K) -> Option<&mut V>;
> unsafe fn get_unchecked(&self, index: K) -> &V;
> unsafe fn get_unchecked_mut(&mut self, index: K) -> &mut V;

I have some doubts, but I will make the necessary changes first